master/usage/configuration/mtu.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Configuring MTU
+---
+
+Depending on the environment Calico is being deployed into it may be
+helpful or even necessary to configure the MTU of the TAP (or veth) that is
+attached to each workload and the tunnel devices if IP-in-IP is enabled.
+
+### Selecting MTU size
+
+Typically the MTU for your workload interfaces should match the optimum MTU
+your network.  If you need IP-in-IP then the MTU size for both the workload
+*and* tunnel interfaces should be 20 bytes less than the optimum MTU for your
+network.  This is due to the extra 20 byte heder that the tunnel will add
+to each packet.
+
+#### Common MTU sizes
+
+| Network MTU | Calico MTU | Calico MTU with IP-in-IP |
+|-------------|------------|--------------------------|
+| 1500 | 1500 | 1480 |
+| 9000 | 9000 | 8980 |
+| 1460 (GCE) | 1460 | 1440 |
+| 9001 (AWS Jumbo) | 9001 | 8981 |
+
+### Setting MTU for workload network interfaces
+
+It is the job of the network plugin to create new interfaces, the current
+major plugins are CNI and libnetwork.  Currently Docker and the Mesos Docker
+Containerizer integration use libnetwork which does *not* support setting MTU.
+CNI which is used by Kubernetes and the Mesos Unified Containerizer support
+configuring the MTU through the CNI configuration file.
+
+#### MTU configuration with CNI
+
+To set the MTU when using CNI the line `"mtu": <MTU size>` must be added to
+the CNI configuration file.
+
+Example CNI configuration
+
+```json
+{
+    "name": "any_name",
+    "cniVersion": "0.1.0",
+    "type": "calico",
+    "mtu": 1480,
+    "ipam": {
+        "type": "calico-ipam"
+    }
+}
+```
+
+> **Note:** The MTU on existing workloads will not be updated with this
+change.  To have all workloads use the new MTU then they must be restarted.
+
+### Setting MTU for tunnel network interfaces
+
+If IP-in-IP is enabled and and the MTU needs to be modifed then this must be
+configured by setting a Felix environment variable or using calicoctl to set
+the proper configuration variable.
+
+#### Setting MTU through Felix Environment variable
+
+Adding the environment variable `FELIX_IPINIPMTU` to the calico/node container
```

Comment:
I would saying *passing in* the environment when *running* the calico/node container